conda_env.yml

```diff
@@ -278,4 +278,4 @@ dependencies:
     - uproot3==3.14.1
     - uproot3-methods==0.10.0
     - widgetsnbextension==3.5.1
-prefix: /beegfs/desy/user/anovak/miniconda3/envs/DLtest
+prefix: /beegfs/desy/user/anovak/miniconda3/envs/DJC2
```

Comment:
I think this line should be removed entirely

Reply:
Good catch. I suppose that gets created to show provenance, but no needed here.